classes/scanner.php

```diff
@@ -38,15 +38,13 @@ function __construct() {
      * @return boolean
      */
     public function is_configured() {
+        // Simply curl the conncheck endpoint and get the status code.
         $curl = new \curl();
         $host = get_config('antivirus_remote', 'scanhost');
         $resp = $curl->get($host . '/conncheck');
         $obj = json_decode($resp);
```

Comment:
do we need to also check if the json is well formed?

Reply:
We could, Id rather that gets exposed as an error in the antivirus handler though via an exception throw